roles/default/kiali-deploy/defaults/main.yml

```
@@ -114,7 +116,9 @@ kiali_defaults:
         type: "none"
         use_kiali_token: false
         username: ""
-      is_core_component: false
+      component_status:
+        is_core: false
+        health_check_url: ""
```

all peer settings should be sorted alphabetically. "h" before "i" :)

```suggestion
        health_check_url: ""
        is_core: false
```